### ISSUE TITLE
Could you show some detail about the training loss implementation

### ISSUE THREAD

**Issue by GarrickZ2** [caiyuanhao1998/PNGAN#1]
Hi there,
I'm wondering how you implement the loss function for Generator and Discriminator respectively. I notice that your overall loss is quite original, but I still hope to know which part will be involved in the Generator's Loss, and which part will be involved in the Discriminator's Loss. 
Thanks for any response!

**Comment by caiyuanhao1998** [comment]
Hi, thanks for your attention,
The loss functions are detailed in Eq. (2), (6), (12), and (13) in our paper.

We use a pre-trained denoiser RID-Net as a regularizer to conduct image domain alignment in Eq. (2)
The loss functions of Generator and Discriminator are given in Eq. (6)
Eq. (12) describes perceptual loss
Eq. (13) is the overall training objective which is a weighted sum of Eq. (2), (6), (12)

Admittedly, the training process is very sophisticated. We are going to do more research and extend our paper to a journal version. I appreciate your kind [re-implementation](https://github.com/GarrickZ2/Image-Denoising) of our PNGAN. I will mention your work in this repo. Thank you very much.
